### DISCUSSION TITLE
Rounded profile image in the main page of blog

### DISCUSSION THREAD

**Discussion by be-favorite** [joelnitta/joelnitta-home#8] [Q&A]
Hi, I'm a person who really love the design in your blog. 

What I want to ask you is, how did you make the image rounded in the main page of blog? 

I'm using the same postcards theme as you: https://www.taemobang.com

As you know, default setting of postcards::trestles theme is squared image.

Please let me know how you make it! thanks :)

**Comment by joelnitta** [comment]
Thanks for the question!

It's quite simple (no R code involved). I just used a PNG file with transparent background. The file is here: https://github.com/joelnitta/joelnitta-home/blob/main/content/images/joel.png

Does that help?

**Reply by be-favorite** [reply]
OMG, that's why there were no differences in _site.yml between yours and mine. just because image is rounded.😂 Thanks for reply!
